conda/_vendor/cpuinfo/__main__.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This file should have been skipped when using the [vendoring](https://github.com/pradyunsg/vendoring) tool that we have configured in https://github.com/conda/conda/blob/8b1635e8f39663220352efc958ae06fb395d8933/pyproject.toml#L27. Could you double-check if this process worked for you?

Reply:
I updated the vendoring by hand initially which included that file.  I have since reverted that commit and used the tool which does not include the `__main__` file.